crates/subspace-farmer-components/src/plotting.rs

```diff
@@ -133,8 +133,8 @@ where
     //  explicitly and, ideally, we need to remove 2x replication
     //  expectation from other places too
     let current_segment_index = farmer_protocol_info.total_pieces.get()
-        / u64::from(farmer_protocol_info.recorded_history_segment_size)
-        / u64::from(farmer_protocol_info.record_size.get())
+        / RecordedHistorySegment::SIZE as u64
```

Comment:
Shouldn't this divide `total_pieces` by `PIECES_IN_SEGMENT`?

Reply:
The code before this PR doesn't make sense to me either, fixed